src/results/chp.jl

```diff
@@ -111,6 +111,7 @@ function add_chp_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict; _n="")
     r["year_one_chp_fuel_cost_before_tax"] = round(value(m[:TotalCHPFuelCosts] / p.pwf_fuel["CHP"]), digits=3)                
 	r["lifecycle_chp_fuel_cost_after_tax"] = round(value(m[:TotalCHPFuelCosts]) * p.s.financial.offtaker_tax_pct, digits=3)
 	#Standby charges and hourly O&M
+	r["lifecycle_chp_standby_cost"] = round(value(m[Symbol("TotalCHPStandbyCharges")]) * p.s.financial.offtaker_tax_pct, digits=0)
```

Comment:
I had removed this output here because it is now included in results/financial.jl (in order to have the full LCC breakdown there). If you think it should also be repeated here, please add "after_tax" to the output name.

Reply:
I hadn't intentionally added it back in (I think this is how the merge of our changes was resolved), but I do think it would be good to have the duplicate here.